test/GeomeanOracle.t.sol

```diff
@@ -198,4 +197,25 @@ contract TestGeomeanOracle is Test, Deployers {
         assertEq(observation.tickCumulative, 13862);
         assertEq(observation.secondsPerLiquidityCumulativeX128, 680564733841876926926749214863536422912);
     }
+
+    function testPermanentLiquidity() public {
+        initializeRouter.initialize(key, SQRT_RATIO_2_1, ZERO_BYTES);
+        geomeanOracle.setTime(3); // advance 2 seconds
+        modifyLiquidityRouter.modifyLiquidity(
+            key,
+            IPoolManager.ModifyLiquidityParams(
+                TickMath.minUsableTick(MAX_TICK_SPACING), TickMath.maxUsableTick(MAX_TICK_SPACING), 1000
+            ),
+            ZERO_BYTES
+        );
+
+        vm.expectRevert();
```

Comment:
would be great to put the expected revert reason in here! like
`OraclePoolMustLockLiquidity.selector`
if thats the expected error